plugins/inputs/exec/README.md

```diff
@@ -34,16 +30,24 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # environment = []
 
   ## Timeout for each command to complete.
-  timeout = "5s"
-
-  ## measurement name suffix (for separating different commands)
-  name_suffix = "_mycollector"
-
-  ## Data format to consume.
-  ## Each data format has its own unique set of configuration options, read
-  ## more about them here:
+  # timeout = "5s"
+
+  ## Measurement name suffix
+  ## Used for separating different commands
+  # name_suffix = ""
+
+  ## Ignore Error Code
+  ## If set to true, a non-zero error code in not considered an error and the
+  ## plugin will continue to parse the output.
+  # ignore_error = false
```

Comment:
Shouldn't we allow the user to specify error codes here? Maybe including a `*` or some special value to signal "all"?

Reply:
That was the direction I originally went with, but then decided if someone wanted to ignore errors, then why make the user research and determine which error codes could come from the command. That could be a long list if they are trying to ignore any error.

Reply:
I'm fine with accepting as is because I cannot come-up with a proper, simple solution. :-)